HCP docs update for IBM Power (#92359)
openshift/openshift-docs: modules/hcp-ibm-power-heterogeneous-nodepools-create-agent-cluster.adoc

// Module included in the following assemblies:
//
// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc

:_mod-docs-content-type: PROCEDURE
[id="hcp-ibm-power-heterogeneous-nodepools-create-agent-cluster_{context}"]
= Create agent cluster

.Procedure

. Create a hosted cluster for heterogeneous node pool
+
Use a multi-architecture release image when creating the cluster to enable support for heterogeneous node pools. Find the latest multi architecture images on the link:https://multi.ocp.releases.ci.openshift.org/[Multi-arch release images] page.

. Export the cluster namepsace.
+
[source,terminal]
----
$ export CLUSTERS_NAMESPACE=<hosted_cluster_namespace>
----

. Export the machine classless inter-domain routing (CIDR) notation.
+
[source,terminal]
----
$ export MACHINE_CIDR=192.168.122.0/24
----
// Typically the namespace is created by the hypershift-operator 
// but agent cluster creation generates a capi-provider role that
// needs the namespace to already exist

> . Run the following command to create the hosted control namespace:

Reviewer: to keep it consistent
```suggestion
. Create the hosted control namespace by running the following command:
```

+
[source,terminal]
----
$ oc create ns ${HOSTED_CONTROL_PLANE_NAMESPACE}
----

> . Run the following command to create the cluster:

Reviewer: ```suggestion
. Create the cluster by running the following command:
```

+
[source,terminal]
----

> $ hypershift create cluster agent \
>     --name=<hosted_cluster_name> \ <1>
>     --pull-secret=<pull_secret_file> \ <2>
>     --agent-namespace=<hosted_control_plane_namespace> \ <3>
>     --base-domain=<basedomain> \ <4>
>     --api-server-address=api.<hosted_cluster_name>.<basedomain> \

Reviewer: Annotations with multiline commands require this so they format correctly in the output
```suggestion
$ hypershift create cluster agent \
    --name=<hosted_cluster_name> \// <1>
    --pull-secret=<pull_secret_file> \// <2>
    --agent-namespace=<hosted_control_plane_namespace> \// <3>
    --base-domain=<basedomain> \// <4>
    --api-server-address=api.<hosted_cluster_name>.<basedomain> \
```

    --release-image=quay.io/openshift-release-dev/ocp-release:<ocp_release> <5>
----
+

> <1> The hosted cluster name.
> <2> The pull secret file path.
> <3> The namespace for the hosted control plane.
> <4> The basedomain for the hosted cluster.
> <5> The current Openshift release version.

Reviewer: ```suggestion
<1> Specify the hosted cluster name.
<2> Specify the pull secret file path.
<3> Specify the namespace for the hosted control plane.
<4> Specify the base domain for the hosted cluster.
<5> Specify the current {product-title} release version.
```

// Module included in the following assemblies:
//
// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc

:_mod-docs-content-type: PROCEDURE
[id="hcp-ibm-power-heterogeneous-nodepools-create-agent-cluster_{context}"]
= Create an agent cluster

An agent cluster is a cluster that is managed and provisioned using an agent-based approach. An agent cluster can utilize heterogeneous node pools, allowing for different types of worker nodes to be used within the same cluster.

.Prerequisites

> * You used a multi-architecture release image to enable support for heterogeneous node pools when creating a hosted cluster. Find the latest multi architecture images on the link:https://multi.ocp.releases.ci.openshift.org/[Multi-arch release images] page.

Reviewer 2: You use a dash between them in the previous sentence: 

```suggestion
* You used a multi-architecture release image to enable support for heterogeneous node pools when creating a hosted cluster. Find the latest multi-architecture images on the link:https://multi.ocp.releases.ci.openshift.org/[Multi-arch release images] page.
```

.Procedure

. Create an environment variable for the cluster namespace by running the following command:
+
[source,terminal]
----
$ export CLUSTERS_NAMESPACE=<hosted_cluster_namespace>
----

. Create an environment variable for the machine classless inter-domain routing (CIDR) notation by running the following command:
+
[source,terminal]
----
$ export MACHINE_CIDR=192.168.122.0/24
----
// Typically the namespace is created by the hypershift-operator 
// but agent cluster creation generates a capi-provider role that
// needs the namespace to already exist

. Create the hosted control namespace by running the following command:
+
[source,terminal]
----
$ oc create ns <hosted_control_plane_namespace>
----

. Create the cluster by running the following command:
+
[source,terminal]
----

> $ hypershift create cluster agent \

Author: @srivastav-abhishek could you please confirm ? thanks.
